### PR TITLE
Updated Documentation and unit test errors

### PR DESCRIPTION
**Pull request by AQuentin2033**
Fixes to documentation and duplicate unit test errors

### CODE REVIEW

File: src/SpatialUnitTests/Euclidean/CoordinateSystemTest.cs [hunk 1]
```diff
@@ -223,7 +223,6 @@ public void SetToAlignCoordinateSystemsTest(string fcss, string tcss)
         [TestCase(X, Y, null)]
         [TestCase(X, Y, "0,0,1")]
         [TestCase("1,-1, 1", "0, 1, 1", null)]
-        [TestCase(X, Y, Z)]
```
Comment (JohanLarsson) [inline src/SpatialUnitTests/Euclidean/CoordinateSystemTest.cs]: Why was this removed?

Comment (AQuentin2033) [reply src/SpatialUnitTests/Euclidean/CoordinateSystemTest.cs]: duplicate

**Comment by JohanLarsson** [comment]
Awesome, thanks!
